outlines/debugging-graphql-queries-with-graphiql.md

```diff
@@ -0,0 +1,69 @@
+# Debugging GraphQl queries with GraphiQL
```

Comment:
This whole blog could be merged with 
Exploring GraphiQL: The In-Browser IDE for GraphQL